HLTrigger/HLTfilters/src/HLTSinglet.cc

```diff
@@ -104,6 +105,7 @@ HLTSinglet<T>::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   desc.add<double>("MinE",-1.0);
   desc.add<double>("MinPt",-1.0);
   desc.add<double>("MinMass",-1.0);
+  desc.add<double>("MinEta",-1.0);
```

Comment:
Hi @silviodonato - MinAbsEta is likely a less confusing name for this parameter in the long term.